### ISSUE TITLE
DiffusionNFT Training on Few-Step Distilled Diffusion Models Becomes Progressively Blurry and Collapses: Instability vs. Loss Mismatch?

### ISSUE THREAD

**Issue by FaltingsA** [NVlabs/DiffusionNFT#45]
基于few-step蒸馏模型进行DiffusionNFT 训练会导致越训越糊，出现崩坏情况，是因为少步情况下的不稳定性还是loss层面的不适配？是否有一些建议来调整参数适配基于蒸馏模型进行diffusionNFT？
When training DiffusionNFT on a few-step distilled model, the outputs become blurrier as training progresses and may even collapse. Is this due to instability inherent to the low-step setting, or a mismatch at the loss/objective level? Do you have any suggestions for tuning hyperparameters to better adapt DiffusionNFT to a distilled diffusion model?

**Comment by zhengkw18** [comment]
DiffusionNFT currently does not support step-distilled models because it is built on the flow matching loss. Adapting the method to distilled models is left for future work.

**Comment by Abecid** [comment]
Training step-distilled methods with vanilla diffusion objective leading to bluriness is mentioned here:
https://arxiv.org/abs/2410.03190
